addons/event/models/event_event.py

```diff
@@ -289,41 +296,64 @@ def _onchange_is_online(self):
-                        for attribute_name in self.env['event.type.ticket']._get_event_ticket_fields_whitelist()
-                    )
-                    all_ticket_values.append(ticket_vals)
-
-                self.event_ticket_ids = [(5, 0, 0)] + [(0, 0, item) for item in all_ticket_values]
+    @api.depends('event_type_id', 'event_type_id.default_registration_min', 'event_type_id.default_registration_max')
+    def _compute_seats_availability(self):
+        for event in self:
+            if event.event_type_id:
+                event.seats_min = event.event_type_id.default_registration_min
+                event.seats_max = event.event_type_id.default_registration_max
+                if event.event_type_id.default_registration_max:
+                    event.seats_availability = 'limited'
+            else:
+                event.seats_min = 0
+                event.seats_max = 0
+                event.seats_availability = 'unlimited'
+
+    @api.depends('event_type_id', 'event_type_id.auto_confirm')
+    def _compute_auto_confirm(self):
+        for event in self:
+            event.auto_confirm = event.event_type_id.auto_confirm if event.event_type_id else False
+
+    @api.depends('event_type_id', 'event_type_id.use_timezone', 'event_type_id.default_timezone')
+    def _compute_date_tz(self):
+        for event in self:
+            event.date_tz = self.env.user.tz or 'UTC'
+            if event.event_type_id and event.event_type_id.use_timezone and event.event_type_id.default_timezone:
+                event.date_tz = event.event_type_id.default_timezone
+
+    @api.depends('event_type_id', 'event_type_id.is_online')
+    def _compute_is_online(self):
+        for event in self:
+            event.is_online = event.event_type_id.is_online if event.event_type_id else False
+
+    @api.depends('event_type_id', 'event_type_id.use_mail_schedule', 'event_type_id.event_type_mail_ids')
+    def _compute_event_mail_ids(self):
+        for event in self:
+            if event.event_type_id:
+                if event.event_type_id.use_mail_schedule and event.event_type_id.event_type_mail_ids:
+                        event.event_mail_ids = [(5, 0, 0)] + [
+                            (0, 0, {
+                                attribute_name: line[attribute_name] if not isinstance(line[attribute_name], models.BaseModel) else line[attribute_name].id
+                                for attribute_name in self.env['event.type.mail']._get_event_mail_fields_whitelist()
+                            })
+                            for line in event.event_type_id.event_type_mail_ids]
+            else:
+                event.event_mail_ids = self.env['event.mail']
+
+    @api.depends('event_type_id', 'event_type_id.use_ticket', 'event_type_id.event_type_ticket_ids')
+    def _compute_event_ticket_ids(self):
+        for event in self:
+            if event.event_type_id:
+                if event.event_type_id.use_ticket:
```

Comment:
```suggestion
            if event.event_type_id and event.event_type_id.use_ticket:
```

Reply:
Hi @MiquelRForgeFlow, I have just made the PR so that I could see a diff while working on it so nothing here is actually reviewable yet.

Reply:
oh, okay :+1: